tools/SetupFlow/DevHome.SetupFlow/Views/LoadingView.xaml

```diff
@@ -32,6 +32,23 @@
                             </Setter.Value>
                         </Setter>
                     </Style>
+                    <!-- Because a data template is an encapsulation layer, dynamic style lookups won't look past the data template.
+                         This means that using a converter to change foreground to a ThemeResource on a textblock does not work.
```

Comment:
"This means that using a converter to change foreground to a ThemeResource on a textblock does not work."

Isn't what we're doing here using a converter? I'm thinking we could actually make a BoolToTextStyleConverter that we can use in other places in the app